@xen-orchestra/lite/src/components/host/network/HostPifsTable.vue

```diff
@@ -160,67 +164,49 @@ import {
 import { noop } from '@vueuse/shared'
 import { computed, ref } from 'vue'
 import { useI18n } from 'vue-i18n'
-import { useRoute } from 'vue-router'
 
-const { records, isReady, hasError } = usePifStore().subscribe()
-const { getByOpaqueRef: getHostOpaqueRef } = useHostStore().subscribe()
-const { getByOpaqueRef: getNetworkOpaqueRef } = useNetworkStore().subscribe()
-const { getPifCarrier } = usePifMetricsStore().subscribe()
+const { pifs } = defineProps<{
+  pifs: XenApiPif[]
+}>()
+
+const { isReady, hasError } = usePifStore().subscribe()
+const { getByOpaqueRef } = useNetworkStore().subscribe()
+const { getPifStatus } = usePifStore().subscribe()
 
 const { t } = useI18n()
 const selectedPifId = useRouteQuery('id')
 const searchQuery = ref('')
 
-const route = useRoute()
-
-const hostId = route.params.uuid as XenApiHost['uuid']
-
-const pifs = computed(() => {
-  return records.value.filter(pif => {
-    const host = getHostOpaqueRef(pif.host)
-
-    return host?.uuid === hostId
-  })
-})
-
 // TODO change to match with network name
 const filteredPifs = computed(() => {
   const searchTerm = searchQuery.value.trim().toLocaleLowerCase()
 
   if (!searchTerm) {
-    return pifs.value
+    return pifs
   }
 
-  return pifs.value.filter(pif =>
-    Object.values(pif).some(value => String(value).toLocaleLowerCase().includes(searchTerm))
-  )
+  return pifs.filter(pif => Object.values(pif).some(value => String(value).toLocaleLowerCase().includes(searchTerm)))
```

Comment:
Can we at least do that for now, to be able to search on network name?

```suggestion
  return pifs.filter(pif =>
    [...Object.values(pif), getNetworkName(pif.network)].some(value =>
      String(value).toLocaleLowerCase().includes(searchTerm)
    )
  )
```